src/test/java/step5/domain/RepsTest.java

```diff
@@ -0,0 +1,56 @@
+package step5.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+
+class RepsTest {
+
+    @Test
+    @DisplayName("Reps 생성 테스트")
+    void Reps_생성_테스트() {
+        int given = 4;
+
+        Reps result = new Reps(given);
+
+        assertThat(result.getReps()).isEqualTo(given);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {0, -1, -100})
+    @DisplayName("0보다 작은 숫자일 때 예외를 던진다")
+    void _0보다_작은_숫자일_때_예외를_던진다(int input) {
+        assertThatIllegalArgumentException().isThrownBy(() -> new Reps(input))
+                .withMessage("0보다 큰 수를 입력해주세요.");
+    }
+
+    @Test
+    @DisplayName("Reps 차감 테스트")
+    void Reps_차감_테스트() {
+        Reps result = new Reps(5);
+
+        result.repsDown();
+
+        assertThat(result.getReps()).usingRecursiveComparison()
+                .isEqualTo(4);
+    }
+
+    @Test
+    @DisplayName("진행할 Reps가 존재할떄 테스트")
```

Comment:
```suggestion
    @DisplayName("isOverZero 메서드는 진행할 수 있는 경우 true를 반환한다.")
```

콘솔에서 DisplayName만보고도 어떤 테스트고 통과가되었는지 확인을 할 수 있으면 테스트가 수백개 이상이 되었을때 하나하나 확인해볼 번거로움이 줄어들겠죠 👍